### DISCUSSION TITLE
Welcome to Fingerprint_TF Discussions!

### DISCUSSION THREAD

**Discussion by JinZhuXing** [JinZhuXing/Fingerprint_TF#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ravi-prakash1907** [comment]
Hi @JinZhuXing,

I found this repository quite interesting.  
I am not clear whether **Fingerprint_TF** is a research based project or not. If in case, it includes a whole research, I would love to go through your published literature; if you don't mind to share either through this repository's documentation or via e-mail.. Moreover, we are doing some research work in biometrics.We found your dataset to be very useful. Hence, I need to discuss some things related to the same.

**Reply by JinZhuXing** [reply]
Hi @ravi-prakash1907 

Thank you for your interest in my project.
I am an engineer who develop Fingerprint recognition engine.
I have already designed OEM based fingerprint recognition engine, and it is used in many devices.
Our company is making Fingerprint devices.

This project, Fingerprint_TF, is my research based project.
OEM based fingerprint engine is not using Deep Learning.
I want to make fingerprint engine using Deep Learning, so I started this project.
But it is not completed yet.
Current published network does not satisfy me.
So I am now preparing new network for training, and after complete this network, I will publish my research paper.

My work is not fast enough, because I have no GPU, so I am using cloud based training, and it is not easy work.
Also I have not enough datasets.
But I will work hard for making perfect engine.

Thank you for your support.
